bin/console

```diff
@@ -7,7 +7,7 @@ require 'dashx'
 # with your gem easier. You can also use a different console, if you like.
 
 # (If you use this, don't forget to add pry to your Gemfile!)
-# require 'pry'
+# require "pry"
```

Comment:
[nit] Can this be changed back to single quotes? (I know this file was auto-generated)